cli/kubectl_kadalu/remove_archived_pv.py

```diff
@@ -0,0 +1,110 @@
+"""
+'remove-archived-pv' subcommand for kubectl-kadalu CLI tool
+"""
+from __future__ import print_function
+import sys
+import json
+import utils
+
+# noqa # pylint: disable=too-many-instance-attributes
+# noqa # pylint: disable=useless-object-inheritance
+# noqa # pylint: disable=too-few-public-methods
+# noqa # pylint: disable=bad-option-value
+def set_args(name, subparsers):
+    """ add arguments to argparser """
+    parser = subparsers.add_parser(name)
+    arg = parser.add_argument
+
+    arg("name", metavar="NAME",
+        help="Specify Storage name to delete archived pvc(s)")
+    arg("--pvc", metavar="PVC", default=None,
+        help="name of archived pvc belonging to specified storage-pool")
+    utils.add_global_flags(parser)
+
+
+def validate(args):
+    """
+    Validate the storage of archived pvcs requested to be delete
+    is present in kadalu configmap or not.
+    Exit if not present.
+    """
+
+    storage_info_data = get_configmap_data(args)
+
```

Comment:
- it's better to add another validation that PVC name if supplied should start with `archived`?

Reply:
Done. Added it in CSI to avoid having checks in two places, if the standalone script is to be run from CSI.